crates/vm/src/wasm/host_env.rs

```diff
@@ -251,6 +251,7 @@ where
             "namada_tx_write" => Function::new_typed_with_env(wasm_store, &env, wrap_tx::_4(host_env::tx_write)),
             "namada_tx_write_temp" => Function::new_typed_with_env(wasm_store, &env, wrap_tx::_4(host_env::tx_write_temp)),
             "namada_tx_yield_value" => Function::new_typed_with_env(wasm_store, &env, wrap_tx::_2(host_env::tx_yield_value)),
+            "namada_tx_has_conversion" => Function::new_typed_with_env(wasm_store, &env, wrap_tx::_2(host_env::tx_has_conversion)),
```

Comment:
there's probably no way around this, but this will necessitate update of all tx and vp wasms (i.e. migration)